sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -406,13 +406,11 @@ export class CallbackFunction<E, R> extends LambdaFunction {
         // Copy over all option values into the function args.  Then overwrite anything we care
         // about with our own values.  This ensures that clients can pass future supported
         // lambda options without us having to know about it.
-        // The default version for Lambda functions now is NodeJS16. As of April 30 2022, Node12
-        // is EOL/ nodeJS 14 is only in maintenance mode so it's best to upgrade to Node16
         const functionArgs = {
             ...args,
             code: code,
             handler: serializedFileNameNoExtension + "." + handlerName,
-            runtime: args.runtime || Runtime.NodeJS16dX,
+            runtime: args.runtime || Runtime.NodeJS18dX,
```

Comment:
Should we take the opportunity to just upgrade to 20x?

Reply:
You are right, https://github.com/pulumi/pulumi-aws/issues/4134 was suggesting 18 as the current LTS but https://endoflife.date/nodejs  suggests 20 is the current LTS so that should be reasonable.